cmd/pulumi-test-language/interface.go

```diff
@@ -769,25 +771,44 @@ func (eng *languageTestServer) RunLanguageTest(
 		return makeTestResponse(fmt.Sprintf("program snapshot validation failed:\n%s", strings.Join(validations, "\n"))), nil
 	}
 
+	project, err := workspace.LoadProject(filepath.Join(projectDir, "Pulumi.yaml"))
+	if err != nil {
+		return makeTestResponse(fmt.Sprintf("load project: %v", err)), nil
+	}
+
+	var projectRawItems map[string]interface{}
+	err = yaml.Unmarshal(project.RawValue(), &projectRawItems)
+	if err != nil {
+		return makeTestResponse(fmt.Sprintf("unmarshal project: %v", err)), nil
+	}
+
+	var projectOptions map[string]interface{}
+	entryPoint := "."
+	if projectRawItems != nil {
+		if options, has := projectRawItems["options"].(map[string]interface{}); has {
+			projectOptions = options
```

Comment:
I believe this part isn't working as expected because if I print `projectOptions` I get:
```
options: { fields: { key: ..., value: { string_value: ... }}}
```